src/replace/include/replace/Replace.h

```diff
@@ -78,7 +78,7 @@ class Replace
     void setDb(odb::dbDatabase* odb);
     void setSta(sta::dbSta* dbSta);
     void setResizer(rsz::Resizer* resizer);
-    void setFastRoute(grt::GlobalRouter* fr);
+    void setGlobalRouter(grt::GlobalRouter* fr);
```

Comment:
too bad replace does not have a clue about initializing the tool. this should not be done with a set function, it should be done in the init function